tools/scripts/renode-test-all.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -x
```

Comment:
Maybe `set -e` in to stop on first error. May require setting a trap to cleanup. Or you could break and set an exit code to call at the end.

Reply:
Removed `-x`. Disagree on `-e`, but I've added exit points on `make` invocations.